### PR TITLE
[API] ShopBillingData as Embedded Channel's Resource 

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
|-----------------|-----
| Branch?         | api-platform-3 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no 
| License         | MIT

### CODE REVIEW

File: tests/Api/Admin/ChannelsTest.php [hunk 1]
```diff
@@ -159,109 +151,37 @@ public function it_updates_an_existing_channel(): void
...
-//        );
-//
-//        $this->assertResponse(
-//            $this->client->getResponse(),
-//            'admin/channel/delete_channel_that_cannot_be_deleted',
-//            Response::HTTP_UNPROCESSABLE_ENTITY,
-//        );
-//    }
+    /** @test */
+    public function it_prevents_single_channel_deletion(): void
```
Comment (NoResponseMate) [inline tests/Api/Admin/ChannelsTest.php]:
```suggestion
    public function it_prevents_deleting_the_only_channel(): void
```
better to revert it, `single` sounds like we cannot delete any channel

File: tests/Api/Admin/ChannelsTest.php [hunk 2]
```diff
@@ -15,111 +15,95 @@
...
+        ]);
+
+        $this->assertResponseCreated('admin/channel/post_channel_response');
     }
 
     /**
      * @test
      *
      * @dataProvider getBlankFieldsData
-     * @dataProvider getTooLongFieldsData
```
Comment (NoResponseMate) [inline tests/Api/Admin/ChannelsTest.php:110]: Why?

Comment (Rafikooo) [reply tests/Api/Admin/ChannelsTest.php:110]: A mistake, I forgot to revert it.

**Review by Wojdylak — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @Rafikooo!

**Pull request merged**